.github/workflows/kernel_test.yml

```diff
@@ -52,34 +52,58 @@ jobs:
         id: boot_test_mb
         run: make run AUTO_TEST=boot ENABLE_KVM=0 BOOT_PROTOCOL=multiboot RELEASE=1
 
+      - name: Make Clean
+        run: make clean
+
```

Comment:
Separate the `make clean` and memory modifications into another commit. As this has nothing to do with `syscall_creat`.

Also this will significantly increase the billable minutes running CI. And it absolutely solves NOTHING in the CI scenario.

Reply:
In my previous PR, there was no action to clean the code, but in github's CI test integration-test, the above test failed. So, I had to put makeclean in this pr in order to pass the test.  

If the problem of insufficient memory can be solved by mounting the hard disk as soon as possible, I also think it is not necessary to do this operation. Cleaning up code is a quick thing though.

Reply:
I still don't know why `make` would make the initramfs larger if not cleaned. If so, the problem with the regression make system would contain serious problems. Please have another issue showing the fact if you can test it (just show `du -h` after a certain amount of makes).

Reply:
The question is very strange indeed. I reproduced him. I'll send you the video.